yapf/yapflib/pytree_unwrapper.py

```diff
@@ -224,6 +224,8 @@ def Visit_async_stmt(self, node):  # pylint: disable=invalid-name
       if pytree_utils.NodeName(child) == 'ASYNC':
         break
     for child in node.children[index].children:
+      if (child.type == grammar_token.NAME and child.value == 'else'):
```

Comment:
I believe you should be able to just use this (please check that `ELSE` is correct):

```
if pytree_utils.NodeName(child) == 'ELSE':
```

Reply:
The node name is `NAME`, and there doesn't seem to be a name for 'ELSE', so I would still have to check the value. Which as far as I understand it would make the line:

```
if (pytree_utils.NodeName(child) == 'NAME' and child.value == 'else'):
```